src/main/java/org/folio/service/inventory/InventoryHoldingManager.java

```diff
@@ -232,8 +234,8 @@ private Future<JsonObject> createHoldingsRecord(String instanceId, String locati
       });
   }
 
-  private Future<String> createHoldingsRecordId(String instanceId, String locationId, RequestContext requestContext) {
-    return createHoldingsRecord(instanceId, locationId, requestContext)
+  public Future<String> createHoldingReturnId(String instanceId, String locationId, RequestContext requestContext) {
```

Comment:
Suggest renaming to `createHoldingAndReturnId` as it was a bit confusing at first glance

Reply:
done